render/detailed_node.go

```diff
@@ -237,20 +238,33 @@ func processOriginTable(nmd report.NodeMetadata) (Table, bool) {
 			rows = append(rows, Row{Key: tuple.human, ValueMajor: val, ValueMinor: ""})
 		}
 	}
+	if addHostTag {
+		rows = append([]Row{{Key: "Host", ValueMajor: report.ExtractHostID(nmd)}}, rows...)
+	}
 
+	title := "Process"
+	var (
+		commFound, pidFound bool
+		name, pid           string
+	)
+	if name, commFound = nmd.Metadata[process.Comm]; commFound {
+		title += ` "` + name + `"`
+	}
+	if pid, pidFound = nmd.Metadata[process.PID]; pidFound {
+		title += " (" + pid + ")"
+	}
```

Comment:
I like the declaration block, but I am not in favor of systematically covering every case when you can do it incrementally. For me it's more compact and easier to read.